### PR TITLE
KAFKA-8974: trim whitespaces in topic

### PR DESCRIPTION
**Pull request by mageshn**
This is a minor change that will improve the UX when the topic list includes whitespace in the `topics` config for a SinkConnector

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java [hunk 1]
```diff
@@ -287,6 +287,7 @@ protected void initializeAndStart() {
 
         if (SinkConnectorConfig.hasTopicsConfig(taskConfig)) {
             String[] topics = taskConfig.get(SinkTask.TOPICS_CONFIG).split(",");
+            Arrays.parallelSetAll(topics, i->topics[i].trim());
```
Comment (wicknicks) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java]: do we really need to process this in parallel? seems like `setAll()` should be simple and clean enough.

Comment (mageshn) [reply connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java]: yeah, that's a good point. I have modified the code per your suggestion.

**Review by wicknicks — APPROVED** [review]
LGTM.

**Review by rhauch — APPROVED** [review]
Thanks, @mageshn. LGTM. 

Valid characters for Kafka topics [are the ASCII alphanumerics, '.', '_', and '-'](https://github.com/apache/kafka/blob/2.3.0/clients/src/main/java/org/apache/kafka/common/internals/Topic.java#L75), so trimming the topic names before assignment makes sense.

File: connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java [hunk 2]
```diff
@@ -287,6 +287,7 @@ protected void initializeAndStart() {
 
         if (SinkConnectorConfig.hasTopicsConfig(taskConfig)) {
             String[] topics = taskConfig.get(SinkTask.TOPICS_CONFIG).split(",");
+            Arrays.setAll(topics, i->topics[i].trim());
```
Comment (rhauch) [inline connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java]:
```suggestion
            Arrays.setAll(topics, i -> topics[i].trim());
```

**Review by rhauch — COMMENTED** [review]
@mageshn, the checkstyle run has failed:
```
'->' is not followed by whitespace. | 290
'->' is not preceded with whitespace. | 290
```

**Pull request merged**